api/helpers/uploadedFileErrorChecks.js

```diff
@@ -291,32 +284,15 @@ function checkTypeSizeAndDuplication(req, file, cb) {
     }
 
     if (errors.length > 0) {
-        addErrorsToSession(req, file.originalname, errors);
+        addErrorsToPage(req, file.originalname, errors);
         preventFileUpload();
     } else {
         allowFileUplaod();
     }
 }
 
-function addErrorsToSession(req, fileName, errors) {
-    const fileNamesWithErrors = req.session.eApp.uploadMessages.errors.map(
-        (error) => error.hasOwnProperty('filename') && error.filename
-    );
-    if (fileNamesWithErrors.includes(fileName)) {
-        fileNamesWithErrors.forEach((fileNameFromSession, idx) => {
-            if (fileNameFromSession === fileName) {
-                req.session.eApp.uploadMessages.errors[idx].errors = [
-                    ...req.session.eApp.uploadMessages.errors[idx].errors,
-                    ...errors,
-                ];
-            }
-        });
-    } else {
-        req.session.eApp.uploadMessages.errors.push({
-            filename: fileName,
-            errors,
-        });
-    }
```

Comment:
This code was here, well the first part of the if statement, because there was an assumption that one file could have multiple errors, so there was a check to add an error to an already existing file and display it to the user, sort of like bullet points.

But I've noticed a file will always usually have one error. And if it does have one error (not a pdf, infected with virus, too large, duplicate), the file is usually removed